src/MTGCore.ClientApp/ClientApp/src/assets/css/Main.css

```diff
@@ -11,4 +11,7 @@
     .modal-leave-active .modal-dialog {
         -webkit-transform: scale(1.1);
         transform: scale(1.1);
-    }
\ No newline at end of file
+    }
+.table {
+  scale: 300%;
+}
```

Comment:
This scale causes the card table to not fit in the window.

![image](https://user-images.githubusercontent.com/17239606/118694779-562b7280-b804-11eb-9a9d-d611fb8a79cb.png)

Also, I think it may be a decent idea to create a new CSS class to separate it from the bootstrap one. Maybe something like: `.table-large` or `.table.large` or whatever